### PR TITLE
Setup error reporting 363

### PR DESCRIPTION
**Pull request by sareg0**
This is a basic first implementation for the exception notifier. I'm not sure what else we would want to receive notifications on apart from exceptions. Contribution and review on this PR is welcome and encouraged 🙏 Many thanks in advance for your thoughful feedback.

### CODE REVIEW

**Comment by lislis** [comment]
As reference and help, I used the same gem here https://github.com/rubymonsters/slam-alphas/pull/198/  @sareg0

**Comment by til** [comment]
And another hint: the exception_notification now supports error grouping to limit the number of mails received in case of an error: https://github.com/smartinez87/exception_notification#error-grouping

It needs to be explicitly turned on, and definitely should be.

File: config/environments/development.rb [hunk 1]
```diff
@@ -28,6 +28,12 @@
...
 
-  config.action_mailer.delivery_method = :test
-
+  config.action_mailer.delivery_method = :letter_opener
+
+  config.middleware.use ExceptionNotification::Rack,
+    email: {
+      email_prefix: "[ERROR] ",
+      sender_address: %{"Rubycorns" <roar@rorganize.it>},
+      exception_recipients: %w[regan.sara@gmail.com]
```
Comment (PragTob) [inline config/environments/development.rb]: maybe change me or remove me? :)

**Comment by lislis** [comment]
party

**Pull request merged**